helper.py

```diff
@@ -331,8 +334,10 @@ def solve_equation(phase_cur_frame,info_dict):
         phase_diff.append(phase_cur_frame[soham]-phase_cur_frame[soham-1])
     Tp=cfg.Tp
     Tc=cfg.Tc
-    L=info_dict[' L'][0]/100
-    r0=info_dict[' R'][0]/100
+    L = info_dict[0]/100
+    r0 = info_dict[1]/100
+    #L=info_dict[' L'][0]/100
+    #r0=info_dict[' R'][0]/100
```

Comment:
Why?

Reply:
It was required for PINN, passing the L_R array instead of info_dict from the merged_data L_R[0] gives L and L_R[1] gives R. Since it is not required for MSE I removed it in recent commit along with other PINN code.